hackerspace_site/templates/home/spacejam.html

```diff
@@ -0,0 +1,189 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>SpaceJam-2021</title>
+    <link rel="stylesheet" href="{% static 'css/spaceJam.css'%}" />
+    <!-- Icon that is visible on the tab -->
+    <link rel="icon" href="../../static/images/icon.png">
+  </head>
+  <body>
+    <div class="main-container">
+      <section class="section1" id="Home">
+        <!-- Header -->
+        <nav>
+          <ul>
+            <!-- <a href="{% url 'home' %}" style="color: white;"> -->
+        
+                <li class="logo"><a href="{% url 'home' %}" class="logo" style="color: white;">&lt;/HackerSpace&gt;</a></li>
+        
+            <!-- </a> -->
+            <div class="links">
+              <a href="#Home" class="navLink"><li class="about">Home</li></a>
+              <a href="#AboutUs" class="navLink"
+                ><li class="about">About Us</li></a
+              >
+              <a href="#Contacts" class="navLink"
+                ><li class="contact">Contacts</li></a
```

Comment:
Maybe replace contacts with sponsors? (unless you want to add contacts there later)